### PR TITLE
Bug fixes and improvements 🧼

### LINKED ISSUE #2: Doesn't work if system language is not English
https://github.com/connorads/meet-kicker/blob/master/src/content.js#L3
`Remove from meeting` is not the same as `Retirer de la réunion`
Language

### LINKED ISSUE #4: Left-clicking the icon on a new tab will show the right-click menu instead of performing the action
Seems to happen if the tab has been created and then we navigate straight to Google Meet.
Workaround to "activate it" is to click on another tab and switch back.

image

### LINKED ISSUE #5: Logo should be grey when not on meet.google.com
image

Think this should be possible somehow ...
image

### LINKED ISSUE #6: Link to Chrome App Store is broken on readme
image

### PR DESCRIPTION
**Pull request by connorads**
- [x] Add error alerts
- [x] Support non-English systems (Fixes #2)
- [x] Update Readme - fix broken image and remove incorrect bonus message (Fixes #6)
- [x] Show page action on tab creation (Fixes #4)
- [x] Only show active page action on meet.google.com (Fixes #5)

### CODE REVIEW

File: src/content.js [hunk 1]
```diff
@@ -1,6 +1,30 @@
+const getAlertText = (errorMessage) => `Meet Kicker 🦶
+
+${errorMessage}
+
+• Is there someone else in the meeting?
+• Do you have permission to remove users?
+
```
Comment (connorads) [inline src/content.js:6]: aka if you try and manually kick someone are you able to?

**Comment by connorads** [comment]
Ugh I wish I squashed this 😢

**Comment by thientrieu12345** [comment]
> Ugh I wish I squashed this 😢

haha😂😂🤣🤣🤣🤣

File: src/content.js (hunk 1 above)
Comment (thientrieu12345) [reply src/content.js:6]: no

Comment (thientrieu12345) [reply src/content.js:6]: i can't kick anyone

Comment (connorads) [reply src/content.js:6]: Do you get an error? What does it say?

**Comment by thientrieu12345** [comment]
it can kick anyone... :((((

Vào Th 6, 5 thg 11, 2021 vào lúc 15:46 Connor Adams <
***@***.***> đã viết:

**Pull request merged**